java/rapidoid/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM maven:3.6-jdk-8 as maven
+WORKDIR /rapidoid
+COPY pom.xml pom.xml
+COPY src src
+RUN mvn compile assembly:single -q
+
+FROM openjdk:11-jre-slim
+WORKDIR /rapidoid
+COPY --from=maven /rapidoid/target/benchmark-0.0.1-SNAPSHOT-jar-with-dependencies.jar app.jar
+CMD ["java", "-server", "-XX:+UseNUMA", "-XX:+UseParallelGC", "-XX:+AggressiveOpts", "-cp", "app.jar", "benchmark.rapidoid.Main", "profiles=production"]
```

Comment:
Could you explain a little bit the purpose of this parameters ?

PS : I think it could be cool to have the same optimization options on all `java` **frameworks** here

Reply:
To be honest I just copied this from somewhere. After a bit of searching I found out the default GC for java 11 is G1, so the setting overriding that to use parallel GC. The agressiveOpts is deprecated in java 11, and useNUMA is taking advantage of machines capable of discriminating differend speeds of access to memory. Works with parallel GC. See https://docs.oracle.com/javase/7/docs/technotes/guides/vm/performance-enhancements-7.html#numa
This kind of fine tuning requires benchmarking to find the sweet spot. What do you think?

Reply:
Honestly, I am not in `java` world, so I'll prefer letting you decide :stuck_out_tongue: 
However, what's important is to make sure
+ spring-boot
+ rapidoid
+ act
use the same tweaks

could you homogenize docker files ?

/cc @greenlaw110

Reply:
Done for rapidoid and spring boot. For Act it is not possible since it requires java 8 runtime (it makes use of sun.misc.Unsafe, which was removed at some point before java 11). Act also has its own opinionated launcher, so I cannot force XX options. Updated the Dockerfile so that it is similar to the others, though.

Reply:
:+1: